recipe/meta.yaml

```diff
@@ -11,27 +11,30 @@ source:
   sha256: 90af1beaa7bf9e4c52db29ec251ec4fd0a8f2cc185d521ad1f88d01b3a6a17e3
 
 build:
-  number: 0
-  skip: true  # [win or py3k]
+  number: 1
+  skip: true  # [win]
   detect_binary_files_with_prefix: true
 
 requirements:
   build:
-    - perl 5.22.2.1
+    - perl
     - pkg-config
-    - harfbuzz 1.7.*
+    - pthread-stubs
```

Comment:
Should this be in `build` or in `host`?

Reply:
It literally provides a single pkg-config file, so it works in `host`. Cf [libxcb #8](https://github.com/conda-forge/libxcb-feedstock/pull/8).